client/foundries_ci.go

```diff
@@ -0,0 +1,47 @@
+package client
+
+import (
+	"encoding/json"
+
+	"github.com/sirupsen/logrus"
+)
+
+type Worker struct {
+	Name           string   `json:"name"`
+	Enlisted       bool     `json:"enlisted"`
+	Online         bool     `json:"online"`
+	Surges         bool     `json:"surges"`
+	HostTags       []string `json:"host_tags"`
+	ConcurrentRuns int      `json:"concurrent_runs"`
+	CpuType        string   `json:"cpu_type"`
+	CpuTotal       int      `json:"cpu_total"`
+	MemTotal       int      `json:"mem_total"`
+	Distro         string   `json:"distro"`
+}
+
+func (a *Api) CiWorkersList() ([]Worker, error) {
+	url := a.serverUrl + "/workers/"
+	type response struct {
+		Data struct {
+			Next    string   `json:"next"`
+			Workers []Worker `json:"workers"`
+		} `json:"data"`
+	}
```

Comment:
I buy that you want this to be private, that's fine.
But, any reason it is not a module level type?

Reply:
a big regret i have with jobserv was this "data" envelope. It serves no value or purpose. 

but - milo now implemented this support in admin.fio. I'm not sure this feature is actually needed any more in fioctl. I'm thinking we just close this for now and keep it in our back-pocket for another day.